tests/render/nointerpolation.hlsl

```diff
@@ -1,6 +1,6 @@
 //TEST(smoke):COMPARE_HLSL_RENDER:
 // WGSL: nointerpolate doesn't work #5625
-//DISABLE_TEST(smoke):COMPARE_HLSL_RENDER: -wgpu
+//TEST(smoke):COMPARE_HLSL_RENDER: -wgpu
```

Comment:
https://github.com/shader-slang/slang/pull/5641 will enable this. I don't think it should work yet.

Reply:
What kind of output do you see? It should be a solid red triangle, I think.